### ISSUE TITLE
LSPosed is not hidden

### ISSUE THREAD

**Issue by snake-4** [snake-4/Zygisk-Assistant#19]
```
/dev/block/dm-42 /apex/com.android.art/bin/dex2oat64 ext4 ro,seclabel,noatime,resgid=1065,errors=panic 0 0
```
The mount above is mounted even if no Xposed modules are injected into the target app.

**Comment by snake-4** [comment]
Fixed in 5c2ba2a

**Comment by snake-4** [comment]
Apparently still doesn't work on some setups.

**Comment by snake-4** [comment]
Fixed in 7cbb41b for all root implementations.
